vllm/attention/backends/utils.py

```diff
@@ -242,6 +253,11 @@ def build(self, seq_lens: List[int], query_lens: List[int],
         seq_start_loc = torch.zeros(seq_lens_tensor.shape[0] + 1,
                                     dtype=torch.int32,
                                     device=device)
+        placeholder_index_maps = {
+            modality: placeholder_map.index_map()
+            for modality, placeholder_map in
+            self.multimodal_placeholder_maps.items()
+        }
```

Comment:
I've suggested changes in other files for the code to be consistent with the code here!

Reply:
Yes, sorry I missed those other ones!